### PR TITLE
Create 2018-05-19-osu-mapping-olympiad-2-taiko.md

### PR DESCRIPTION
**Pull request by osuChaos**
### Description

Create newspost for 2nd osu! mapping olympiad contest (taiko gamemode)

### Status

- [x] Pending Review

### CODE REVIEW

File: news/2018-05-19-osu-mapping-olympiad-2-taiko.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+After the entries are judged and the results are finalized, the top three entries will receive the following prizes:
+
+- **1st place**: 3 months of osu!supporter, 6 Olympiad Points (Results in 'Mapping Olympian' Badge), *Elite Mapper* title
+- **2nd place**: 2 months of osu!supporter, 4 Olympiad Points
+- **3rd place**: 1 month of osu!supporter, 2 Olympiad Points
+
+For Olympiad contests, we will be using a point system to award the 'Mapping Olympian' badge. This means that top placing results across multiple contests will grant you the opportunity to be awarded a badge even if you do not win a particular contest. Every 6 points earned will grant you a badge level, with incrementation on the badge for each level achieved. The expected baseline for points rewards will be 6,4, and 2 for the top 3 entries respectively, but the specific rewards will vary per contest based on expected effort and challenge for the participants. 
+
+You will have 21 days from the date of this post to submit your entries. Download the pre-timed .osz [here](https://assets.ppy.sh/artists/7/osz/Thaehan%20-%20Bwa%20!.osz).
```
Comment (fajar13k) [inline news/2018-05-19-osu-mapping-olympiad-2-taiko.md]: Never use `here` as the link text. Instead, you can highlight all the text as a link
> [Download the pre-timed .osz](https://assets.ppy.sh/artists/7/osz/Thaehan%20-%20Bwa%20!.osz)

File: news/2018-05-19-osu-mapping-olympiad-2-taiko.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+All rules and important information, as well as a submission deadline countdown, are available at the [contest page](https://osu.ppy.sh/community/contests/65) — refer to it if you need any further clarification. In any case, we’d love to see what you can do, so give us all you’ve got. Good luck, and have fun!
```
Comment (fajar13k) [inline news/2018-05-19-osu-mapping-olympiad-2-taiko.md]: - Another optional simplify
> [contest page](/community/contests/65)

- And another contraction
`we'd` love -> `we would` love
`you've` got -> `you have` got

File: news/2018-05-19-osu-mapping-olympiad-2-taiko.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+
+All entries will be judged by the following criteria:
+
+- **Flow and Structure (10pts)**: How well are the difficulties internally designed? Do they play comfortably and cohesively?
+- **Relevance (10pts)**: How well does the technical execution of the map fit the song in terms of rhythm, emphasis, and intensity?
+- **Difficulty Spread (10pts)**: Is the difficulty progression well executed? Are gameplay elements introduced throughout the spread in a reasonable manner?
+- **Creativity (10pts)**: How well does the submission stand out using creative, fitting elements?
+- **Judge impression (10pts)**: What do judges think about the submission as a whole?
+
+**We are looking for capable judges! If you are interested, please make a post in the [Judge Call Thread](https://osu.ppy.sh/community/forums/topics/747545) or send a message to [hikikochan](https://osu.ppy.sh/users/6512678). Please be aware that we can only accept people who have the resources (knowledge, time, reliability) to properly judge the entries.**
```
Comment (fajar13k) [inline news/2018-05-19-osu-mapping-olympiad-2-taiko.md]: These are optional simplify:
> [Judge Call Thread](/community/forums/topics/747545)
> [hikikochan](/users/6512678)

File: news/2018-05-19-osu-mapping-olympiad-2-taiko.md [hunk 4]
```diff
@@ -0,0 +1,61 @@
+---
+layout: post
+title: "osu! Mapping Olympiad #2: osu!taiko"
+date: 2018-05-19 13:00:00 +0000
+---
+
+As osu!’s iteration of the Mapping Olympiad is wrapping up, it’s time to bring another mode into the spotlight: osu!taiko! For the second contest in this series, we bring you an opportunity to become osu!taiko’s first ever Mapping Olympian and also claim the title of Elite Mapper!
```
Comment (fajar13k) [inline news/2018-05-19-osu-mapping-olympiad-2-taiko.md]: Contractions. Do
>it is time to bring another . . . .

File: news/2018-05-19-osu-mapping-olympiad-2-taiko.md [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+
+[![](/wiki/shared/news/banners/osu_mapping_olympiad.jpg)](https://osu.ppy.sh/community/contests/65)
+
+It’s certainly been a while since osu!taiko’s had an official mapping contest, so we’re excited to be bringing this contest to everyone, complete with prizes and an opportunity to win the coveted Elite Mapper title!
```
Comment (fajar13k) [inline news/2018-05-19-osu-mapping-olympiad-2-taiko.md]: Another contractions:
> It is certainly . . . .
> we are excited to . . . .

File: news/2018-05-19-osu-mapping-olympiad-2-taiko.md [hunk 6]
```diff
@@ -0,0 +1,61 @@
...
+
+Just to reiterate, the goal of the osu! Mapping Olympiad is to provide mapping contests to all game modes on a regular basis. If you aren’t a taiko mapper, stay on the lookout for more opportunities to come – there will be a wide variety of contest formats in the future, from traditional full spreads to rapid-fire speedmapping and everything in between.
```
Comment (fajar13k) [inline news/2018-05-19-osu-mapping-olympiad-2-taiko.md]: Contractions:
`aren't` -> `are not`

File: news/2018-05-19-osu-mapping-olympiad-2-taiko.md [hunk 7]
```diff
@@ -0,0 +1,61 @@
...
+
+## Song
+
+To start off your Olympiad journey as a drum master, we have chosen a song by featured artist **Loki/Thaehan**!
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/7/previews/86.mp3" type="audio/mpeg">
+</audio>
+
+*Bwa !*, an upbeat electronic track, is filled to the brim with colorful synthesizer melodies and a variety of rhythmic patterns. Its chiptune style allows for a wide range of creative interpretations, so we’re excited to see the sets you create out of this song!
```
Comment (fajar13k) [inline news/2018-05-19-osu-mapping-olympiad-2-taiko.md]: so `we're` excited . . . -> so `we are` excited

File: news/2018-05-19-osu-mapping-olympiad-2-taiko.md [hunk 8]
```diff
@@ -0,0 +1,61 @@
...
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/7/previews/86.mp3" type="audio/mpeg">
+</audio>
+
+*Bwa !*, an upbeat electronic track, is filled to the brim with colorful synthesizer melodies and a variety of rhythmic patterns. Its chiptune style allows for a wide range of creative interpretations, so we are excited to see the sets you create out of this song!
+
+## Rules
+
+- This contest is for the **osu!taiko** gamemode only.
+- Each entry **must follow the Ranking Criteria**. Failure to adhere to the Ranking Criteria may result in penalties ranging from point deduction to total disqualification, depending on severity.
```
Comment (MegaApplePi) [inline news/2018-05-19-osu-mapping-olympiad-2-taiko.md]: link to the RC?
```diff
- follow the Ranking Criteria**. Failure
+ follow the [Ranking Criteria](/wiki/Ranking_Criteria)**. Failure
```

File: news/2018-05-19-osu-mapping-olympiad-2-taiko.md [hunk 9]
```diff
@@ -0,0 +1,61 @@
...
+
+## Prizes
+
+After the entries are judged and the results are finalized, the top three entries will receive the following prizes:
+
+- **1st place**: 3 months of osu!supporter, 6 Olympiad Points (Results in 'Mapping Olympian' Badge), *Elite Mapper* title
+- **2nd place**: 2 months of osu!supporter, 4 Olympiad Points
+- **3rd place**: 1 month of osu!supporter, 2 Olympiad Points
+
+For Olympiad contests, we will be using a point system to award the 'Mapping Olympian' badge. This means that top placing results across multiple contests will grant you the opportunity to be awarded a badge even if you do not win a particular contest. Every 6 points earned will grant you a badge level, with incrementation on the badge for each level achieved. The expected baseline for points rewards will be 6,4, and 2 for the top 3 entries respectively, but the specific rewards will vary per contest based on expected effort and challenge for the participants.
```
Comment (MegaApplePi) [inline news/2018-05-19-osu-mapping-olympiad-2-taiko.md]: missing a space after `6,`
```diff
- be 6,4, and
+ be 6, 4, and
```

File: news/2018-05-20-osu-mapping-olympiad-2-taiko.md [hunk 10]
```diff
@@ -1,7 +1,7 @@
 ---
 layout: post
 title: "osu! Mapping Olympiad #2: osu!taiko"
-date: 2018-05-19 13:00:00 +0000
+date: 2018-05-20 13:00:00 +0000
 ---
 
 As osu!’s iteration of the Mapping Olympiad is wrapping up, it is time to bring another mode into the spotlight: osu!taiko! For the second contest in this series, we bring you an opportunity to become osu!taiko’s first ever Mapping Olympian and also claim the title of Elite Mapper!
```
Comment (MegaApplePi) [inline news/2018-05-20-osu-mapping-olympiad-2-taiko.md]: change `’` to `'`

**Pull request merged**